spark/src/test/resources/tpcds-plan-stability/approved-plans-v1_4-spark4_0/q1/explain.txt

```diff
@@ -31,9 +31,9 @@ TakeOrderedAndProject (40)
       :     :                                   :     +- CometScan parquet spark_catalog.default.store_returns (11)
       :     :                                   +- ReusedExchange (14)
       :     +- BroadcastExchange (31)
-      :        +- * ColumnarToRow (30)
-      :           +- CometProject (29)
-      :              +- CometFilter (28)
+      :        +- * Project (30)
+      :           +- * Filter (29)
+      :              +- * ColumnarToRow (28)
```

Comment:
Hmmm do you know why CometFilter is no longer activated by any chance?

Reply:
No. But just want to restore CI as soon as possible. We can investigate it later.